pkg/limits/limits.go

```diff
@@ -17,4 +22,68 @@ const (
 	TTLMax                = 30 * time.Minute
 	ClusterAffiliatesMax  = 1024
 	AffiliateEndpointsMax = 64
+	RequestsPerSecondMax  = 5
 )
+
+// IPRateLimiter type.
+type IPRateLimiter struct {
+	IPs        map[string]*rate.Limiter
```

Comment:
it might make sense to make all these fields private (starting with lowercase letter), so that they are not available from the outside for direct access